app/controllers/proposals_controller.rb

```diff
@@ -49,6 +49,13 @@ def create
     # by default.
     @event.speakers = [current_user]
     @event.submitter = current_user
+
+    if Track.find_by(id: params[:track_id]).try(:cfp_active) == false
```

Comment:
We want to load the track that corresponds to the track_id in the parameters hash. I don't think that cancancan can load it, because there is no track_id in the url for it to use and we don't want it to load the track that might be associated with the event

Reply:
That's true our params have [:event][:track_id], but then what you are writing in this line is wrong. Did you try if this is working on your browser?